src/pages/community-champions.js

```diff
@@ -122,6 +142,71 @@ const WhatItMeansToBeAChampionText = ({
     );
 };
 
+const LeadByExampleContainer = styled('div')`
+    background-color: ${({ theme }) => theme.colorMap.greyDarkThree};
+    border-radius: ${size.xsmall};
+    padding: ${LEAD_BY_EXAMPLE_CONTAINER_PADDING};
+    margin: ${WHAT_IT_MEANS_TO_BE_A_CHAMPION_ROW_GAP} 0 ${size.xlarge} 0;
+    @media ${screenSize.upToLarge} {
+        padding: ${size.mediumLarge};
+        margin: ${size.large} 0;
+    }
+`;
+
+const LeadByExampleItemContainer = styled('div')`
+    display: grid;
+    grid-column-gap: ${size.mediumLarge};
+    grid-template-columns: ${LEAD_BY_EXAMPLE_ICON_WIDTH} auto;
+    @media ${screenSize.upToLarge} {
+        grid-template-columns: ${LEAD_BY_EXAMPLE_ICON_MOBILE_WIDTH} auto;
+    }
+`;
+
+const LeadByExampleSectionTitle = styled('div')`
```

Comment:
Would one of the options from `text.js` make sense here? 

Where possible using heading tags (h1, h2) and paragraph tags (p) help also with setting a good DOM structure. text should. generally be wrapped in a text-related HTML tag.

[This concept is called semantic elements in HTML and is definitely worth checking into here!](https://www.w3schools.com/html/html5_semantic_elements.asp)

Reply:
spoke to Mike about this, and we decided to create an H6 component, so I will use H6 and style that. 🙂